tslint.json

```diff
@@ -1,18 +1,17 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-      "tslint:recommended",
-      "tslint-react",
-      "tslint-eslint-rules",
-      "tslint-config-prettier"
-    ],
-    "linterOptions": {
-      "exclude": [
-          "node_modules/**"
-      ]
-    },
-    "jsRules": {},
-    "rules": {
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:recommended",
+    "tslint-react",
+    "tslint-eslint-rules",
+    "tslint-config-prettier"
+  ],
+  "linterOptions": {
+    "exclude": ["node_modules/**"]
+  },
+  "jsRules": {},
+  "rules": {
+    "ban-types": false,
```

Comment:
`Function` was previously a disallowed type and `Function` is needed in order to declare args as invokeable.

as a side note, this project should ditch tslint in favor of eslint, as tslint has been abandoned.

Reply:
agreed